### PR TITLE
Update latest cuda info

### PR DESCRIPTION
**Pull request by andyneff**
Updated with info up to 12.6.1

### CODE REVIEW

File: linux/cuda_info.bsh [hunk 1]
```diff
@@ -505,15 +505,16 @@ function cuda_capabilities()
...
       CUDA_ARCHES=(50 52 53 60 61 62 70 72 75 80 86 87 89 90a 90)
       # The 30s are gone, and there's no longer any deprecated? (50 was undeprecated?)
       CUDA_DEPRECATED=()
       ;;
 
-    # 12.4*) ?
+    # 12.7*) ?
     #   # For a while now, nvcc --help appears to have the complete list of arches finally.
+    #   echo -n 'CUDA_ARCHES=('; nvcc --help | sed -n '/^--gpu-architecture\|^--gpu-code/,/^ *$/p' | sed -n '/^ *Allowed/,/^$/p' | tr ',' '\n' | sed -E "s| *'(.*)'\\.?|\\1|; /^$/d" | grep ^sm | sort -u | sed 's|sm_||' | tr '\n' ' ' | sed 's| *$||'; echo ')'; echo -n 'CUDA_CODES=('; nvcc --help | sed -n '/^--gpu-architecture\|^--gpu-code/,/^ *$/p' | sed -n '/^ *Allowed/,/^$/p' | tr ',' '\n' | sed -E "s| *'(.*)'\\.?|\\1|; /^$/d" | grep ^compute | sort -u | sed 's|compute_||' | tr '\n' ' ' | sed 's| *$||'; echo ')'; echo -n 'CUDA_LTOS=('; nvcc --help | sed -n '/^--gpu-architecture\|^--gpu-code/,/^ *$/p' | sed -n '/^ *Allowed/,/^$/p' | tr ',' '\n' | sed -E "s| *'(.*)'\\.?|\\1|; /^$/d" | grep ^lto | sort -u | sed 's|lto_||' | tr '\n' ' ' | sed 's| *$||'; echo ')'
```
Comment (scott-vsi) [inline linux/cuda_info.bsh:517]: OMG

File: linux/cuda_info.bsh [hunk 2]
```diff
@@ -720,29 +721,57 @@ function cuda_capabilities()
...
+      versions=(560.28.03 560.76)
+      ;;
+    # 12.6.1|12.6.68)
+    12.*)
+      versions=(560.35.03 560.94)
+      ;;
+
     # E.g. https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#id4
-    # 12.2*)
+    # 12.7*|{nvcc)
```
Comment (scott-vsi) [inline linux/cuda_info.bsh:774]: Did you want a `{` here?

**Review by scott-vsi — APPROVED** [review]

File: linux/cuda_info.bsh (hunk 1 above)
Comment (andyneff) [reply linux/cuda_info.bsh:517]: I've done this so many times, I just wanted something to copy and paste

File: linux/cuda_info.bsh (hunk 2 above)
Comment (andyneff) [reply linux/cuda_info.bsh:774]:
```suggestion
    # 12.7*|{nvcc version here})
```

**Pull request merged**